pkg/utils/utils.go

```diff
@@ -152,7 +152,7 @@ func RetryNWithBackoff(backoff Backoff, n int, fn func() error) error {
 // If the number of tries is exhausted, the last error will be returned.
 func RetryNWithBackoffCtx(ctx context.Context, backoff Backoff, n int, fn func() error) error {
 	var err error
-	RetryWithBackoffCtx(ctx, backoff, func() error {
+	_ = RetryWithBackoffCtx(ctx, backoff, func() error {
```

Comment:
What about erroring if it fails to retry?

Reply:
I'll revert this. I think one issue is that we don't even use this code except in the tests. I'd much rather import the code from the official AWS go SDK once we actually use a retryer:

https://github.com/aws/aws-sdk-go/blob/master/aws/client/default_retryer.go